tests/wifi.py

```diff
@@ -7,3 +7,5 @@ class WifiScan(rootfs_boot.RootFSBootTest):
 
     def runTest(self):
         wlan.scan()
+        scan_output = wlan.scan()
```

Comment:
do you need to run  wlan.scan() twice?

Reply:
Not sure why this shows twice. The code which I have shows scan.output only once. Will verify and commit it once again.